e2e/page-objects/bugReporting.po.ts

```diff
@@ -17,22 +23,16 @@ export class BugReportingPage {
    * @returns true if a unique Bug-Report is present, false otherwise.
    */
   async isBugReportPresent({ title, severityLabel, bugTypeLabel }: { title: string; severityLabel?: string; bugTypeLabel?: string }) {
```

Comment:
Is there any reason for object destructuring in this method call? Can't we just do something like 

```ts
async isBugReportPresent(title: string, severityLabel?: string, bugTypeLabel?: string) {
}
```
instead?

Reply:
The main purpose is to be able to use named parameters like so:

```ts
    const isBugReportCorrectlyCreated: boolean = await bugReportingPage.isBugReportPresent({
      title: testIssueCreationTitle,
      severityLabel: 'Medium',
      bugTypeLabel: 'DocumentationBug'
    });
```

But i agree this method signature is hard to read. I've modified it to make it look cleaner